lib/runner.js

```diff
@@ -167,10 +168,13 @@ const runner = async options => {
   const db = Db(options.dbClient || options.databaseUrl, log);
   try {
     if (options.schema) {
+      const schemas = getSchemas(options.schema);
       if (options.createSchema) {
-        await db.query(`CREATE SCHEMA IF NOT EXISTS "${options.schema}"`);
+        await db.query(`CREATE SCHEMA IF NOT EXISTS "${schemas[0]}"`);
```

Comment:
So what if there are more schemas in the array?
I guess we should have a for-loop here and call the `CREATE SCHEMA IF NOT EXISTS` queries for all of them?

Reply:
From the perspective I'm at, I would still use .schema if there is one you want to connect. I liked the prior operation completely, but the hitch is only search path.

set .schema and createSchema mostly unchanged, but if there is a searchPath, use this to instead of the .schema, i.e. never call SET SCHEMA ... 

Just a thought.

Reply:
> So what if there are more schemas in the array?
> I guess we should have a for-loop here and call the `CREATE SCHEMA IF NOT EXISTS` queries for all of them?

I think most traditional SQL examples I've seen (tenants as schemas, and more complex models) will have the primary schema in the front, this would most likely be the only one. The way it is coded should work for the issue's use case.

Reply:
Trying to create all probably makes more sense, will probably prevent some questions in the future and will do no harm.